great_expectations/execution_engine/execution_engine.py

```diff
@@ -416,10 +416,9 @@ def resolve_metrics(  # noqa: C901 - 16
         if len(metric_fn_bundle) > 0:
             try:
                 # an engine-specific way of computing metrics together
-                # NOTE: DH 20220328: This is where we can introduce the Batch Metrics Store (BMS)
```

Comment:
@donaldheppner Please note that this location will not work for BMS as sometimes partially-computed metrics are produced here.  A better candidate would be surrounding the call to `ValidationGraph.MetricsCalculator.resolve_validation_graph()` -- happy to discuss.  Thank you.